docs/tutorials/maps.ipynb

```diff
@@ -849,19 +1123,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "rc_params = {\"figure.figsize\": (12, 5.4), \"font.size\": 12}\n",
-    "with plt.rc_context(rc=rc_params):\n",
-    "    smoothed = m_3fhl_gc.smooth(width=0.2 * u.deg, kernel=\"gauss\")\n",
-    "    smoothed.plot(stretch=\"sqrt\", add_cbar=True, vmax=4);"
+    "filename = \"$GAMMAPY_DATA/fermi-3fhl-gc/fermi-3fhl-gc-counts.fits.gz\"\n",
+    "m_3fhl_gc = Map.read(filename)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Interactive Plotting \n",
-    "\n",
-    "For maps with non-spatial dimensions the `Map` object features an interactive plotting method, that works in jupyter notebooks only (Note: it requires the package `ipywidgets` to be installed). We first read a small example cutout from the Fermi Galactic diffuse model and display the data cube by calling `.plot_interactive()`:"
+    "After reading the map we can now plot it on the screen by calling the `.plot()` method:"
```

Comment:
Since this is an API tutorial, can be useful to mention explicitly that `.plot` works for maps with only spatial axis (or only a single bin for the non-spatial axes).